src/main/java/com/cburch/logisim/fpga/fpgagui/FPGACommanderBase.java

```diff
@@ -320,7 +321,8 @@ protected boolean DownLoadDesign(boolean generateOnly, boolean downloadOnly, Str
 					+ HDLPaths[ScriptPath] + File.separator,
 					RootSheet.getNetList(), MyMappableResources,
 					MyBoardInformation, Entities, Behaviors,
-					AppPreferences.HDL_Type.get())) {
+					AppPreferences.HDL_Type.get())
+					&!generateOnly) {
```

Comment:
Do you really intend a bit-wise AND operation here instead of a conditional AND operation? Additionally, I would put a space between `&` and `!` to separate the operators from each other.